### ISSUE TITLE
Use Published Library With Multiplatform Project

### ISSUE THREAD

**Issue by hfhbd** [cy6erGn0m/kotlinx-uuid#10]
Hey, 
I created a simple Multiplatform project, using this UUID library in commonMain
````kotlin
// gradle.build.kts
sourceSets {
    commonMain {
        dependencies {
            implementation("org.jetbrains.kotlinx.experimental:kotlinx-uuid-core:0.0.1")
    }  
}
````
Branch: https://github.com/hfhbd/ComposeTodo/tree/simple
Commit: https://github.com/hfhbd/ComposeTodo/pull/4/commits/e4c5395be0d878a62891701826c74ef859155ec7 and the CI

However, in each target, only this UUID library is not found by Gradle. You still need to include the explicit library for each source set.
````kotlin
sourceSets {
    commonMain {
        dependencies {
            implementation("org.jetbrains.kotlinx.experimental:kotlinx-uuid-core:0.0.1")
        }
    }
    val jsMain by getting {
        dependencies {
            implementation("org.jetbrains.kotlinx.experimental:kotlinx-uuid-core-js:0.0.1")
        }
    }
    val iosMain by getting {
        dependencies {
        }
    }
    val iosArm64Main by getting {
        dependencies {
            implementation("org.jetbrains.kotlinx.experimental:kotlinx-uuid-core-iosarm64:0.0.1")
        }
    }
    val iosX64Main by getting {
        dependencies {
            implementation("org.jetbrains.kotlinx.experimental:kotlinx-uuid-core-iosx64:0.0.1")
        }
    }
    val jvmMain by getting {
        dependencies {
            implementation("org.jetbrains.kotlinx.experimental:kotlinx-uuid-core-jvm:0.0.1")
        }
    }   
}
````
Workaround: https://github.com/hfhbd/ComposeTodo/pull/4/commits/e40928edb49f1093f89c9aeac1fbe4476dcd1f78 and the CI

@cy6erGn0m Do you know why this problem exists? Is something of the published library still missing?

**Comment by cy6erGn0m** [comment]
Looks like somethig is wrong with module metadata or pom files that should have some placeholder.

**Comment by cy6erGn0m** [comment]
The following fixes your issue. Gradle doesn't want to pick up metadata for some reason:

```kotlin
repositories {
    maven {
        setUrl("https://dl.bintray.com/cy6ergn0m/uuid")
        metadataSources { gradleMetadata() } // this does the trick!
    }
}
```

**Comment by cy6erGn0m** [comment]
From this https://docs.gradle.org/current/userguide/publishing_gradle_module_metadata.html#sub:interactions-other-build-tools

> the `pom.xml` or `ivy.xml` file will contain a marker comment which tells Gradle that Gradle Module Metadata exists for this module

So I've lost the marker comment for some reason

**Comment by hfhbd** [comment]
> The following fixes your issue. Gradle doesn't want to pick up metadata for some reason:
> 
> ```kotlin
> repositories {
>     maven {
>         setUrl("https://dl.bintray.com/cy6ergn0m/uuid")
>         metadataSources { gradleMetadata() } // this does the trick!
>     }
> }
> ```

Thank you! This is working.

**Comment by cy6erGn0m** [comment]
Fixed in 0.0.2
